File: cases/taint/go/infeasible-branch-positive/infeasible_branch_positive.go
package dataflowbench

func dfb_source() int { // DFB-SOURCE: infeasible-branch-input
	return 1
}

func dfb_sink(value int) {} // DFB-SINK: infeasible-branch-sink

func run() {
	value := 0
	if true {
		value = dfb_source() // DFB-WITNESS: feasible-tainted-branch
	}
	dfb_sink(value)
}

File: cases/taint/csharp/infeasible-branch-positive/InfeasibleBranchPositive.cs
namespace DataFlowBench;

static class InfeasibleBranchPositive
{
    static int dfb_source() // DFB-SOURCE: infeasible-branch-input
    {
        return 1;
    }

    static void dfb_sink(int value) { } // DFB-SINK: infeasible-branch-sink

    static void Run()
    {
        int value = 0;
        if (true)
        {
            value = dfb_source(); // DFB-WITNESS: feasible-tainted-branch
        }
        dfb_sink(value);
    }
}
